src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -29,6 +35,23 @@ public CommandBox(CommandExecutor commandExecutor) {
         this.commandExecutor = commandExecutor;
         // calls #setStyleToDefault() whenever there is a change to the text of the command box.
         commandTextField.textProperty().addListener((unused1, unused2, unused3) -> setStyleToDefault());
+        commandTextField.setOnKeyTyped(new EventHandler<KeyEvent>() {
```

Comment:
Think EventHandler not working. nt sure if its cux of the commandTextField. trying to use the `scene` way instead? but can't identify it in the `MainApp`

Reply:
have you tried `javafx.scene.input.KeyCombination`